README.md

```diff
@@ -1,16 +1,17 @@
-# Question Set Player
+# Quiz Frontend
 
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 [![codecov](https://codecov.io/gh/avantifellows/question-set-player/branch/main/graph/badge.svg)](https://codecov.io/gh/avantifellows/question-set-player)
```

Comment:
codecov is still pulling from question-set-player repo.
This is a new repo or is there a way to rename the existing repo?